src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -517,11 +526,11 @@ class ShenandoahHeap : public CollectedHeap {
   void update_heap_references(bool concurrent);
   // Final update region states
   void update_heap_region_states(bool concurrent);
-  void rebuild_free_set(bool concurrent);
 
   void rendezvous_threads();
   void recycle_trash();
 public:
+  void rebuild_free_set(bool concurrent);
```

Comment:
Please document the method's public specification so a caller is aware as to what they should expect the method to do.